django/utils/version.py

```diff
@@ -76,6 +76,8 @@ def get_git_changeset():
     This value isn't guaranteed to be unique, but collisions are very unlikely,
     so it's sufficient for generating the development version numbers.
     """
+    if '__file__' not in globals():
```

Comment:
Can we add a small comment explaining this, for future reference?

Reply:
GitHub isn't marking this as outdated now, but I'm hoping that 7846f42 is responsive to your request.